src/sqlparser.rs

```diff
@@ -1037,6 +1040,16 @@ impl Parser {
                 Token::NationalStringLiteral(ref s) => {
                     Ok(Value::NationalStringLiteral(s.to_string()))
                 }
+                Token::HexStringLiteral(ref s) => {
+                    if let Some(bad) = s.chars().find(|c| !c.is_ascii_hexdigit()) {
```

Comment:
[The spec](https://jakewheat.github.io/sql-overview/sql-2011-foundation-grammar.html#binary-string-literal) is more lenient, allowing whitespace as well, as far as I can see. Perhaps leave validation to the consumers?

Reply:
Sure thing. Done.